script/path.js

```diff
@@ -230,6 +224,16 @@ var Path = {
 		Path.updateBagSpace(currentBagCapacity);
 
 	},
+	
+	outfit_call: function(i) {
+		var child = $(this);
+		if(child.attr('id').indexOf('outfit_row_') === 0) {
+			var cName = child.children('.row_key').text();
+			if(cName < lk) {
+				curPrev = child.attr('id');
```

Comment:
curPrev is undefined here, as it was previously coming from the closure scope of the inline function.